Implement some cells to find the n fastest laps of all sessions

tetourne opened the pull request:
I implemented some cell in the notebook fast_lap_analysis.ipynb in order to find the n fastest laps.
This pull request adds these cells to the notebook

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
**Once this PR has been reviewed and has the lgtm label**, please assign durandom for approval by writing `/assign @durandom` in a comment. For more information see:[The Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=b4mad%2Fracing).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/b4mad/racing/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["durandom"]} -->

sesheta commented:
Hi @tetourne. Thanks for your PR.

I'm waiting for a [b4mad](https://github.com/orgs/b4mad/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/b4mad/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=b4mad%2Fracing).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

goern commented:
/ok-to-test

sesheta commented:
@tetourne: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
aicoe-ci/prow/pre-commit | a82ba7c06e63c5b68c9ef477b7062b858492ec28 | [link](https://prow.operate-first.cloud/view/s3/prow/prow-logs/pr-logs/pull/b4mad_racing/118/pre-commit/1597910985048330240) | true | `/test pre-commit`

[Full PR test history](https://prow.operate-first.cloud/pr-history?org=b4mad&repo=racing&pr=118). [Your PR dashboard](https://prow.operate-first.cloud/pr?query=is%3Apr%20state%3Aopen%20author%3Atetourne). Please help us and [open an issue](https://github.com/b4mad/racing/issues?q=is:issue+is:open) when you hit one in your PR.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

sesheta commented:
@tetourne: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
aicoe-ci/prow/pre-commit | be4eec8cf2ea81bb3c7909b309b5ef5095ca35ed | [link](https://prow.operate-first.cloud/view/s3/prow/prow-logs/pr-logs/pull/b4mad_racing/118/pre-commit/1597976688438808576) | true | `/test pre-commit`

[Full PR test history](https://prow.operate-first.cloud/pr-history?org=b4mad&repo=racing&pr=118). [Your PR dashboard](https://prow.operate-first.cloud/pr?query=is%3Apr%20state%3Aopen%20author%3Atetourne). Please help us and [open an issue](https://github.com/b4mad/racing/issues?q=is:issue+is:open) when you hit one in your PR.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

tetourne commented:
I did some modifications. I added some try / except when reading data, since some dataframes were not accessible.
I tested the jupyter notebook that I modified on my laptop and it works properly.
Could you test it again ?

sesheta commented:
@tetourne: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
aicoe-ci/prow/pre-commit | be4eec8cf2ea81bb3c7909b309b5ef5095ca35ed | [link](https://prow.operate-first.cloud/view/s3/prow/prow-logs/pr-logs/pull/b4mad_racing/118/pre-commit/1597976688438808576) | true | `/test pre-commit`
aicoe-ci/prow/kubeval-validation | dd515be9df403928faf656405a102d1eabc522aa | [link](https://prow.operate-first.cloud/view/s3/prow/prow-logs/pr-logs/pull/b4mad_racing/118/kubeval-validation/1598011780326494208) | true | `/test kubeval-validation`

[Full PR test history](https://prow.operate-first.cloud/pr-history?org=b4mad&repo=racing&pr=118). [Your PR dashboard](https://prow.operate-first.cloud/pr?query=is%3Apr%20state%3Aopen%20author%3Atetourne). Please help us and [open an issue](https://github.com/b4mad/racing/issues?q=is:issue+is:open) when you hit one in your PR.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

sesheta commented:
@tetourne: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
aicoe-ci/prow/pre-commit | dd515be9df403928faf656405a102d1eabc522aa | [link](https://prow.operate-first.cloud/view/s3/prow/prow-logs/pr-logs/pull/b4mad_racing/118/pre-commit/1598011780343271424) | true | `/test pre-commit`
aicoe-ci/prow/kubeval-validation | 0ba61b0a6beb3e7b8b6e03721a6d8eae98013a31 | [link](https://prow.operate-first.cloud/view/s3/prow/prow-logs/pr-logs/pull/b4mad_racing/118/kubeval-validation/1598035247595786240) | true | `/test kubeval-validation`

[Full PR test history](https://prow.operate-first.cloud/pr-history?org=b4mad&repo=racing&pr=118). [Your PR dashboard](https://prow.operate-first.cloud/pr?query=is%3Apr%20state%3Aopen%20author%3Atetourne). Please help us and [open an issue](https://github.com/b4mad/racing/issues?q=is:issue+is:open) when you hit one in your PR.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

sesheta commented:
@tetourne: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
aicoe-ci/prow/kubeval-validation | 0ba61b0a6beb3e7b8b6e03721a6d8eae98013a31 | [link](https://prow.operate-first.cloud/view/s3/prow/prow-logs/pr-logs/pull/b4mad_racing/118/kubeval-validation/1598035247595786240) | true | `/test kubeval-validation`
aicoe-ci/prow/pre-commit | 0ba61b0a6beb3e7b8b6e03721a6d8eae98013a31 | [link](https://prow.operate-first.cloud/view/s3/prow/prow-logs/pr-logs/pull/b4mad_racing/118/pre-commit/1598035248430452736) | true | `/test pre-commit`

[Full PR test history](https://prow.operate-first.cloud/pr-history?org=b4mad&repo=racing&pr=118). [Your PR dashboard](https://prow.operate-first.cloud/pr?query=is%3Apr%20state%3Aopen%20author%3Atetourne). Please help us and [open an issue](https://github.com/b4mad/racing/issues?q=is:issue+is:open) when you hit one in your PR.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

tetourne commented:
I'm closing the pull request since I'm still doing modifications on it.
I'll do another PR when the work is complete.
